web/src/composables/useContentHighlighting.ts

```
@@ -47,11 +47,11 @@ export const useContentHighlighting = () => {
   const highlightContent = (content: string, language?: string): string => {
     try {
       let highlighted: string;
-      
+
       // Check if content contains ANSI escape sequences
       // eslint-disable-next-line no-control-regex
       const hasAnsiSequences = /\x1b\[/.test(content);
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)